### PR TITLE
Support of ELV USB-WDE1

### PR DESCRIPTION
**Pull request by lampi87**
Plugin to support weather data receiver from ELV: http://www.elv.at/usb-wetterdaten-empfaenger-usb-wde1-komplettbausatz-1.html

### CODE REVIEW

**Comment by t-mon** [comment]
Since guhd version 0.7.7 the plugin API changed. The major changes were translations and ParamType have now uuids. See in-line comments for more details:

**Comment by t-mon** [comment]
Note: please merge the current master to make sure everything is up to date.

**Comment by lampi87** [comment]
All issues fixed.
Thx for the hint regarding memory leak. Now only one bridge is allowed, as it does not make sense to add more than one receiver

File: plugins/deviceplugins/usbwde/devicepluginusbwde.json [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+                            "unit": "Percentage",
+                            "defaultValue": 0,
+                            "index": 2,
+                            "eventTypeName": "humidity changed"
+                        },
+                        {
+                            "id": "8e058c13-b5bf-4558-b24e-e3197ba7a9de",
+                            "idName": "windStrength",
+                            "name": "wind strength",
+                            "type": "double",
```
Comment (t-mon) [inline plugins/deviceplugins/usbwde/devicepluginusbwde.json]: Is there maybe a unit? https://developers.guh.io/typeutils-h.html#types-unit

File: plugins/deviceplugins/usbwde/devicepluginusbwde.cpp [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+        }
+        // Loop through 8 possible sensor channels
+        for (int i = 1; i < 9; i++) {
+            if (!parts.at(2 + i).isEmpty()) {
+                // Create new device if it does not exist
+                if (!m_deviceList.contains(i)) {
+                    createNewSensor(i);
+                } else {
+                    Device* device = m_deviceList.value(i);
+                    device->setStateValue(temperatureStateTypeId, german.toDouble(parts.at(2+i), &ok));
```
Comment (t-mon) [inline plugins/deviceplugins/usbwde/devicepluginusbwde.cpp]: If you don't check the `ok`, you could leave it away or just use `0`.

File: debian/control [hunk 3]
```diff
@@ -17,6 +17,7 @@ Build-Depends: debhelper (>= 9.0.0),
                qttools5-dev-tools,
                qtconnectivity5-dev,
                libqt5websockets5-dev,
+               libqt5serialport5-dev,
```
Comment (t-mon) [inline debian/control]: Please also add the the `libqt5serialport5` to the guh-plugins package.

**Comment by t-mon** [comment]
Jenkins, ok to test.

**Comment by t-mon** [comment]
Jenkins, retest this please.

**Comment by lampi87** [comment]
Changes done

**Comment by t-mon** [comment]
devicepluginusbwde.cpp:88:14: error: unused variable 'ok' [-Werror=unused-variable]

**Pull request merged**